picr22-enne-1-voistlust/segment_module/Segment.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,6 @@
+segment.c
```

Comment:
Unneeded file. add to git ignore and remove.

Reply:
didnt use any files in picr22-enne-1-voistlust so removed it